lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -159,6 +159,79 @@ static bool TagIsRecordType(dw_tag_t tag) {
   }
 }
 
+/// Get the object parameter DIE if one exists, otherwise returns
+/// a default DWARFDIE. If \c containing_decl_ctx is not a valid
+/// C++ declaration context for class methods, assume no object
+/// parameter exists for the given \c subprogram.
+static DWARFDIE
+GetCXXObjectParameter(const DWARFDIE &subprogram,
+                      clang::DeclContext const &containing_decl_ctx) {
```

Comment:
```suggestion
                      const clang::DeclContext  &containing_decl_ctx) {
```

Inconsistent placement of const (through the patch). I think this is the usual order we use.